src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,44 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+    @Test
+    @DisplayName("요구사항 1 : split")
+    void splitTest() {
+        //given
+        String[] result = "1,2".split(",");
+        // when & then
+        assertThat(result).contains("1","2");
+
+        //given
+        String[] result2 = "1".split(",");
+        // when & then
+        assertThat(result2).contains("1");
```

Comment:
```suggestion
        String[] result = "1".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?